### ISSUE TITLE
el form

### ISSUE THREAD

**Issue by FefeGalaxy** [Lalalau09/desafio-final-m3#1]
esta muy bueno, pero yo le pondría un media query en el form para cuando agrandas la pantalla

**Comment by Lalalau09** [comment]
fefe! vos decis de que se agrande el form no?

**Comment by FefeGalaxy** [comment]
Claro! Osea me refiero a los inputs del form pero después de eso, está todo genial



Enviado desde Outlook<http://aka.ms/weboutlook>

________________________________
De: Laura Bissoni ***@***.***>
Enviado: martes, 26 de julio de 2022 13:37
Para: Lalalau09/desafio-final-m3 ***@***.***>
Cc: FefeGalaxy ***@***.***>; Author ***@***.***>
Asunto: Re: [Lalalau09/desafio-final-m3] el form (Issue #1)


fefe! vos decis de que se agrande el form no?

**Comment by Lalalau09** [comment]
Gracias! Voy a cambiarlo, aunque ya mande el desafio
